modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,264 @@
+If you don't already have one, you can refer to xref:operator::install-kubernetes.adoc[this guide] for instructions.
+* A Couchbase Server RBAC user with application access privileges.
+Refer to xref:getting-started.adoc#creating-an-rbac-user[this section] for instructions.
+
+== Deploying a Sync Gateway Cluster
+
+The Sync Gateway nodes in a cluster are homogeneous in configuration with two exceptions.
+
+* Importer node: under xref:shared-bucket-access.adoc[convergence/shared bucket access], it is recommended that one node handles document import processing.
+For high availability, you can configure more than one node in your cluster to be an import node, though it is strongly discouraged for all Sync Gateway nodes in the cluster to be configured for import processing.
+The configuration of the import Sync Gateway node is slightly different (see xref:config-properties.adoc#databases-foo_db-import_docs[databases.$db.import_docs]).
+* Replicator node: if you are using inter-cluster replication using sg-replicate then there will be one designated replicator node whose configuration is different than the rest of the nodes.
+
+The following sections cover the steps to deploy a cluster of regular Sync Gateway nodes.
+The configuration and deployment controller files will differ if you are deploying a cluster of import nodes or a single replicator node.
+
+For example, to deploy a cluster with 4 regular nodes, 1 import node and 1 replicator node you would run the steps below 3 times, each time with a different configuration and deployment controller file.
+
+=== Configuration file
+
+In this section, you will create the Sync Gateway configuration file and store it in a secret location.
+
+. Create a new file called *sgw-config.json* with the following and replace the `server` key with the addressable Couchbase Server hostname.
++
+[source,json]
+----
+{
+  "logging": {
+    "console": {
+      "log_keys": ["*"]
+    }
+  },
+  "databases": {
+    "db": {
+      "server": "http://cb-example-0000.cb-example:8091", // <1>
+      "bucket": "default",
+      "username": "admin", // <2>
+      "password": "password",
+      "users": { "GUEST": { "disabled": false, "admin_channels": ["*"] } },
+      "allow_conflicts": false,
+      "revs_limit": 20
+    }
+  }
+}
+----
+<1> The server key should point to the Couchbase Server cluster.
+This would typically be of the form `http://CB_SERVER_POD.CB_SERVER_SERVICE_NAME:8091`.
+<2> The username/password keys should match what was setup in the <<pre-requisites, pre-requisites>> section when you configured the Sync Gateway RBAC user.
+. You will use a https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/[Kubernetes configMap] to pass the configuration file to Sync Gateway on launch.
+However, since the Sync Gateway contains sensitive information, it is recommended that you create a secret from the config file and pass that to the Sync Gateway.
```

Comment:
It seems a little confusing to discuss configmap AND secret, since while they're similar, you can't create a secret from the configmap.

Perhaps reference this in a way that recommends a secret (and shows the steps below), but then say "you could also use a configmap if you are not concerned about the extra security the secret adds"?

Reply:
Agreed. Added a clarification to previous line as well
```suggestion
Alternatively, you could use https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/[Kubernetes configMap] if you are not concerned about the security. However, since the Sync Gateway contains sensitive information, it is recommended that you create a secret from the config file and pass that to the Sync Gateway.
```

Reply:
👍